financial-connections/src/main/java/com/stripe/android/financialconnections/features/MarkdownParser.kt

```diff
@@ -25,49 +17,4 @@ internal object MarkdownParser {
         }
         return newText
     }
-
-    internal fun toHtml(pane: ConsentPane): ConsentPane = ConsentPane(
```

Comment:
This is not needed anymore. Markdown strings are converted to Html via `MarkdownToHtmlSerializer`.